pkg/systemd/host.js

```diff
@@ -361,9 +362,11 @@ PageServer.prototype = {
         });
 
         function refresh_pmlogger_state() {
-            if (!pmlogger_service.exists)
+            if (!pmlogger_service.exists) {
+                $('#system-information-enable-pcp').show();
```

Comment:
This should first check if a `pcp` package is actually available. Our packagekit library has `check_missing_packages()` for that, which should be suitable. We don't want to show this on debian-stable, as it would just fail.

Reply:
I was thinking that this is determined statically, per distro, when building.  E.g., on Fedoras we install nfs-utils for NFS client support, on Debian we install nfs-common.  On other distros that haven't made a choice in their packaging, we don't offer to install anything.

`check_missing_packages` is used internally by the install dialog to figure out what the dependencies are and how much needs to be downloaded.  There is no point in running it before opening the dialog, I would say.

Reply:
OK, so a new `configure` option, like for NFS? Makes sense.